spec/controllers/feedbacks_controller_spec.rb

```diff
@@ -52,7 +52,7 @@ def valid_session
 
     describe "when the user already has existing NAMED feedback" do
       before(:each) do
-        feedback = FactoryGirl.create(:feedback, :review => @review, :user => @user, :user_string => "John Doe")
+        feedback = create(:feedback, :review => @review, :user => @user, :user_string => "John Doe")
```

Comment:
Useless assignment to variable - `feedback`.<br>Line is too long. [100/80]<br>Use the new Ruby 1.9 hash syntax.